### PR TITLE
Add NER experiments

### PR DESCRIPTION
**Pull request by thomashacker**
## This `PR` includes all NER experiments

**It adds three new variables in the `project.yml`**
- `ner_config` (config_tok2vec, config_pretraining, config_transformer): Controls which config should be used when training, assembling, and evaluating
- `gpu_id`: Whether GPU or CPU should be used when training/evaluating
- `pretrained_weights`: Filename of the pretrained weight .bin file

**Add new configs for NER**
Added config for tok2vec, pretraining and transformer + configs for assembling the NER models to one pipeline

**Add new commands**
- For every label, I created a separate train & evaluate command (e.g train-ner-i for the label INTERVENTIONS)
- Commands for pretraining: Preprocessing the data and pretraining the weights
- Assemble command for the NER
- Evaluation command that evaluates both Spancat and NER and compares some KPI's

**Transfer-Entities component**
To assemble the different NER models to one pipeline, I've added a custom component that moves `doc.ents` to `doc.spans`

### CODE REVIEW

File: experimental/README.md
Comment (ljvmiranda921) [inline experimental/README.md:7]: Let's remove the diff here! Just so that we won't have stray diffs in other parts of the repo! :D

File: experimental/ner_spancat_compare/scripts/create_pretraining_data.py
Comment (ljvmiranda921) [inline experimental/ner_spancat_compare/scripts/create_pretraining_data.py:8]: Let's run black on this! :D

File: experimental/ner_spancat_compare/scripts/preprocess.py
Comment (ljvmiranda921) [inline experimental/ner_spancat_compare/scripts/preprocess.py:152]: Thanks for adding this part!

**Review by ljvmiranda921 — COMMENTED** [review]
Thanks @thomashacker ! Just added a few formatting comments 👍 
Before we merge, let's run the `black` formatter so that it's consistent with everything :)

File: experimental/ner_spancat_compare/scripts/preprocess.py
Comment (thomashacker) [reply experimental/ner_spancat_compare/scripts/preprocess.py:152]: I thought it might help to get a better insight into the data and the distribution across the different labels 😄

**Review by ljvmiranda921 — APPROVED** [review]

**Comment by ljvmiranda921** [comment]
Ack that's weird, I cannot merge it. It says "Only those with write access to this repository can merge pull requests"

**Comment by svlandeg** [comment]
> Ack that's weird, I cannot merge it. It says "Only those with write access to this repository can merge pull requests"

The permissions have been updated - can you try again? :-)

**Pull request merged**
